settings.gradle

```diff
@@ -1,4 +1,19 @@
 include ':solanaj'
 include ':app'
-include ':ui-kit'
-rootProject.name = "P2P Wallet"
\ No newline at end of file
+rootProject.name = "P2P Wallet"
+addSubfolderProjects("p2p-wallet-ui-app/ui-kit")
```

Comment:
I don't think the `p2p-wallet-ui-app` is a good name for that repo. It is actually ui kit repository with ui kit library itself with the sample

Reply:
May be the `android-ui-kit`